presto-main/src/main/java/com/facebook/presto/operator/aggregation/noisyaggregation/NoisyCountGaussianColumnAggregationUtils.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.noisyaggregation;
+
+import java.util.Random;
+
+public class NoisyCountGaussianColumnAggregationUtils
+{
+    private NoisyCountGaussianColumnAggregationUtils()
+    {
+    }
+
+    public static long computeNoisyCount(long trueCount, double noiseSdv, Random random)
```

Comment:
I think Sdv here stands for standard deviation, but everywhere else we call this noiseScale, so call it noiseScale here to.  (Relatedly, we generally avoid abbreviations like "Sdv" in variable names)

Reply:
Yeah, you are right